draft-ietf-cose-hash-envelope.md

```diff
@@ -77,24 +76,23 @@ The terms COSE, CDDL, and EDN are defined in {{-COSE}}, {{-CDDL}}, {{-EDN}} resp
 
 # Header Parameters
 
-To represent a hash of a payload, the following headers are defined:
+This document specifies the following new header parameters commonly used alongside hashes to identify resources:
 
 TBD_1:
   : the hash algorithm used to produce the payload.
 
 TBD_2:
-  : the content type of the bytes that were hashed to produce the payload.
+  : the content type of the bytes that were hashed (preimage) to produce the payload.
 
 TBD_3:
-  : an identifier enabling a verifier to retrieve the bytes which were hashed to produce the payload.
+  : an identifier enabling retrieval of the original resource (preimage) identified by the payload.
 
 # Hash Envelope CDDL
 
 ~~~ cddl
 Hash_Envelope_Protected_Header = {
     ? &(alg: 1) => int,
-    ? &(typ: 16) => uint / tstr
```

Comment:
why is typ 16 removed?

Reply:
Its not needed in this document.
There are a bunch of other header parameters that are also not needed here, which can be found at:

https://www.iana.org/assignments/cose/cose.xhtml#header-parameters (kid, crit, etc)...